aiinfra-cluster/variables.tf

```diff
@@ -87,6 +89,11 @@ variable "instance_image" {
     family  = string,
     project = string
   })
+  default = {
+    family  = "tf-latest-gpu-gvnic-debian-10"
+    name    = null
+    project = "deeplearning-platform-release"
+  }
```

Comment:
should this go with the slurm default [here](https://github.com/GoogleCloudPlatform/ai-infra-cluster-provisioning/blob/develop/aiinfra-cluster/main.tf#L92)?

Reply:
that is correct. Updated in latest iteration.